core/src/main/java/com/graphhopper/routing/util/parsers/BMWeightAtoBTrackedParser.java

```diff
@@ -0,0 +1,24 @@
+package com.graphhopper.routing.util.parsers;
+
+import com.graphhopper.reader.ReaderWay;
+import com.graphhopper.routing.ev.IntEncodedValue;
+import com.graphhopper.routing.util.parsers.helpers.OSMValueExtractor;
+import com.graphhopper.storage.IntsRef;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class BMWeightAtoBTrackedParser implements TagParser {
```

Comment:
I think we only need one parser-class since they implement exactly the same logic just with different variable names eg:
```
public class BMWeightParser implements TagParser {

    private final IntEncodedValue weightEnc;

    public BMWeightParser(IntEncodedValue weight) {
        weightEnc = weight;
    }

    @Override
    public IntsRef handleWayTags(IntsRef edgeFlags, ReaderWay way, IntsRef relationFlags) {
        return edgeFlags;
    }
}
```

Reply:
Updated as requested